scripts/prepare-release.sh

```diff
@@ -2,30 +2,50 @@
 set -e
 set -o pipefail
 
-PREVIOUS_VERSION=$(grep integreatly-operator deploy/olm-catalog/integreatly-operator/integreatly-operator.package.yaml | awk -F v '{print $2}')
+if [[ -z "$TYPE" ]]; then
+  TYPE="integreatly-operator"
+fi
+
+PREVIOUS_VERSION=$(grep $TYPE deploy/olm-catalog/$TYPE/$TYPE.package.yaml | awk -F v '{print $2}') || echo "No previous version"
+
+if [[ -z "$ORG" ]]; then
+  ORG="integreatly"
+else
+  ORG="$ORG"
+fi
 
 create_new_csv() {
-  operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name integreatly-operator --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
+  if [[ -z "$PREVIOUS_VERSION" ]]
+    then
+      operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name "$TYPE" --csv-channel=rhmi --update-crds
+    else
+      operator-sdk generate csv --csv-version "$VERSION" --default-channel --operator-name "$TYPE" --csv-channel=rhmi --update-crds --from-version "$PREVIOUS_VERSION"
```

Comment:
I'm not sure but I think we are checking if there isn't an entry in $PREVIOUS_VERSION but then using it?

Reply:
The `-z` is checking if a string has a length of zero, so we are only using the `PREVIOUS_VERSION` in the `else` which means it has been set.